### ISSUE TITLE
Ensure the `target` ETL pipeline uses most up-to-date tractability data file

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1800]
**Describe the bug**

The `21.09.5` ETL run used an older tractability file — [tractability_buckets_new_version.tsv](http://ftp.ebi.ac.uk/pub/databases/opentargets/platform/21.09/input/target-inputs/tractability/). This file was specifically created based on the `21.06` release tractability data in order to develop the `target` ETL and the new tractability API response schema.

**Observed behaviour**

The use of the older tractability file means that some assessments are incorrect.

For example, the tractability data for [KRAS (ENSG00000133703)](https://platform.opentargets.org/target/ENSG00000133703) is missing the Advanced Clinical assessment in the small molecule pipeline.

![Screenshot 2021-10-01 at 16 27 05](https://user-images.githubusercontent.com/7490258/135653886-c920c631-55be-44a3-b599-0a10eb0e2818.png)

The assessment is missing despite the fact that we have Known Drugs data for KRAS. 

Note: this is not an upstream data issue as the file submitted by ChEMBL — [tractability_buckets_2021-08-25.tsv](https://storage.googleapis.com/otar001-core/Tractability/21.08/tractability_buckets_2021-08-25.tsv) — contains the correct assessment for KRAS.

**Expected behaviour**

The production ETL run for the `21.11` release must use the most up-to-date tractability file provided by ChEMBL and must be in sync with the CHEMBL database version used to generate the evidence.

**To Reproduce**

Steps to reproduce the behaviour:
1. Visit the [KRAS (ENSG00000133703) profile page](https://platform.opentargets.org/target/ENSG00000133703)
2. Click on "Tractability"
3. Compare the lack of clinical assessments in the small molecule workflow against the availability of data in the Known Drugs section

**Comment by cmalangone** [comment]
@JarrodBaker and @cmalangone 21.11

**Comment by andrewhercules** [comment]
Ticket closed as the `21.11` release uses the most up-to-date tractability data file provided by the ChEMBL team. Also, ChEMBL have confirmed that the data is correct based on the pipeline's use of external libraries. 

For our records, below is the conversation I had with ChEMBL regarding FAU (ENSG00000149806), CALCA (ENSG00000110680), and SOD1 (ENSG00000142168).

My email to ChEMBL:
```
CALCA (ENSG00000110680)

I think this target is missing an assessment for `AB_B1_Approved Drug`. 

There are three antibodies where the MoA targets the Calcitonin gene-related peptide protein family, of which CALCA is a member. These drugs are Galcanezumab (CHEMBL3707328), Fremanezumab (CHEMBL4297756), and Eptinezumab (CHEMBL3833320). 

The tractability assessment for the other member of the same protein family, CALCB (ENSG00000175868), contains an assessment for `AB_B1_Approved Drug`. Yet on the ChEMBL website, the respective compound pages do not specify the binding subunit of the family. In cases like this, the tractability assessment usually applies the same assessment to all members of the complex or family - for example, all 50 members of the Mitochondrial complex I (NADH dehydrogenase) receive an `SM_B1_Approved Drug` assessment due to the approval status of Metformin (CHEMBL1431).

FAU (ENSG00000149806)

I think this target is missing an `OC_B2_Advanced Clinical` and `SM_B1_Approved Drug` assessment.

This target is part of the 80S Ribosome protein complex and so I think it’s a similar issue to the one mentioned above for CALCA and the assessments have not been applied to all complex members. 

In this case, Ataluren (CHEMBL256997) is the small molecule that has been approved. Similarly, ELX-02 (CHEMBL4297744) and MT-3724 (CHEMBL4297789) are the two drugs with an “unknown” drug type that target the protein complex. 

SOD1 (ENSG00000142168)

This target is missing an `OC_B2_Advanced Clinical` assessment, but I wonder if this is an intended omission as the drug targets the SOD1 mRNA?

There is a drug — Tofersen (CHEMBL3833346) — and the curated MoA identifies SOD1 mRNA as the target. This drug has been in Phase III trials for amyotrophic lateral sclerosis and Paula has submitted three evidence strings with Clinical Trial records. 
```

The response from Harris in ChEMBL:

```
Paula and I had a look in your concerns and we tried to investigate them. After having a look at Melanie’s tractability code, she uses ‘mygene’ python package to get the uniprot accession codes. That being said for the first two cases, i.e. ENSG00000110680 and ENSG00000149806 the uniprot codes are different from what we’ve got in ChEMBL and when querying it to populated the respective columns, i.e. AB_B1_Approved Drug, SM_B1_Approved Drug, and OC_B2_Advanced Clinical no results are returned. For these cases, for ENSG00000110680 the family members in ChEMBL are P06881 and P10092 but not P1258. Similarly for ENSG00000149806 has multiple uniprot accessions and P35544 is not among them.

For the third case, ENSG00000142168, the data in the table are correct. In this case, the target is not the protein but the mRNA which for ChEMBL are different targets. Thus, when filtering with the uniprot id we don’t get any rows back.
```
